File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0;
        int j=1;
        int k=1;

        while (j<nums.length){
            if(nums[i]!=nums[j]){
                nums[k]=nums[j];
                i=j;
                k++;

            }
            j++;
        }
        return k;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):

        i=0 #last unique ele
        j=1 #find new unique ele
        k=1 #write index
        n=len(nums)
        while j<n:
            if nums[i]!=nums[j]:
                nums[k]=nums[j]
                i=j
                k+=1
            j+=1
        return k